[2025-03-01T07:36:04.352+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_etl_dag.load_to_postgres_file scheduled__2025-03-01T07:33:00+00:00 [queued]>
[2025-03-01T07:36:04.360+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_etl_dag.load_to_postgres_file scheduled__2025-03-01T07:33:00+00:00 [queued]>
[2025-03-01T07:36:04.361+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-03-01T07:36:04.375+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): load_to_postgres_file> on 2025-03-01 07:33:00+00:00
[2025-03-01T07:36:04.383+0000] {standard_task_runner.py:60} INFO - Started process 4366 to run task
[2025-03-01T07:36:04.385+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'crypto_etl_dag', 'load_to_postgres_file', 'scheduled__2025-03-01T07:33:00+00:00', '--job-id', '377', '--raw', '--subdir', 'DAGS_FOLDER/crypto_etl_dag.py', '--cfg-path', '/tmp/tmprf60s5kh']
[2025-03-01T07:36:04.388+0000] {standard_task_runner.py:88} INFO - Job 377: Subtask load_to_postgres_file
[2025-03-01T07:36:04.440+0000] {task_command.py:423} INFO - Running <TaskInstance: crypto_etl_dag.load_to_postgres_file scheduled__2025-03-01T07:33:00+00:00 [running]> on host 335ace54b661
[2025-03-01T07:36:04.527+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_etl_dag' AIRFLOW_CTX_TASK_ID='load_to_postgres_file' AIRFLOW_CTX_EXECUTION_DATE='2025-03-01T07:33:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-01T07:33:00+00:00'
[2025-03-01T07:36:04.530+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-01T07:36:04.532+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/dags/scripts/load_to_postgres.py']
[2025-03-01T07:36:04.549+0000] {subprocess.py:86} INFO - Output:
[2025-03-01T07:36:05.789+0000] {subprocess.py:93} INFO - [INFO] Đã tải file '2025/03/01/14-36-02.csv' từ MinIO
[2025-03-01T07:36:05.789+0000] {subprocess.py:93} INFO - [INFO] Đã nạp 10 bản ghi vào Data Warehouse thành công.
[2025-03-01T07:36:05.790+0000] {subprocess.py:93} INFO - [INFO] Quá trình ETL hoàn tất. Đã xử lý 1 file mới từ MinIO.
[2025-03-01T07:36:05.866+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-03-01T07:36:05.889+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=crypto_etl_dag, task_id=load_to_postgres_file, execution_date=20250301T073300, start_date=20250301T073604, end_date=20250301T073605
[2025-03-01T07:36:05.926+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-01T07:36:05.945+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
